VaticanBAV/et/BAVet157.xml

```diff
@@ -0,0 +1,233 @@
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0003BAV"/>
+                        <collection>Aethiopici</collection>
+                        <idno>Aeth. 157</idno>
+                    </msIdentifier>
+                    <msContents>
+                        
+                        <msItem xml:id="ms_i1">
+                            <locus from="3r" to="76v"/>
+                            <title type="complete" ref="LIT2314senafe"> The Beauty of the Creation</title>
+                            <explicit xml:lang="am">ድል፡ መነሣቱ፤ በመስቀል፡ ነውና፤ ምነው፡ በዕለተ፡ ዓርብ፡ ወረደ፤ ቢሉ። ኋላ፡ ፍጹም፡ ጥፍአቱ፡ በዕለተ፡ ዓርብ፡ ነውና፤ 
+                                ተፈጸመ፡ ስነ፡ ፍጥረት፤ በሰላመ፤ እግዚአብሔር፡ ዘሎቱ፡ ስብሐት፤ ወጸጋሁ፤ ተሀሉ፤ ምስለ<supplied reason="omitted">፡</supplied> ኃጥእ፡</explicit>
+                            <textLang mainLang="am" otherLangs="gez"/>
+                            </msItem>
+                        
+                        <msItem xml:id="ms_i2">
+                            <locus from="77r" to="77v"/>
+                            <title type="incomplete" ref="LIT2839RepCh123">Malkǝʾ-hymn to Jesus </title>
+                            <note>The text ends abruptly with the repetition of the opening words: <q><sic>ሰለም</sic>፡ ለዝክረ<supplied reason="omitted">፡</supplied> ስምከ<supplied reason="omitted">፡</supplied> <choice><corr></corr><sic> ስም</sic></choice><supplied reason="omitted">፡</supplied> መሃላ፡ ዘ<gap reason="omitted"></gap></q></note>
+                            <textLang mainLang="gez"/>   
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i3">
+                            <locus from="77v" to="79r"/>
+                            <title type="complete" ref="NAR0004comput"></title>
+                            <incipit xml:lang="am">እምፍጥረተ፡ ዓለም፡ <choice><sic>አስከ፡</sic><corr>እስከ</corr></choice>፡ ኅልቀተ፡ ዓለም፡ ፯ሽህ፡ ከ፱፻ከ፳ዓመት፡ ነው፡ በዕለት፡ ቢገቡት፡ ፫፻፷፭ይሆናል፡</incipit>
+                            <explicit xml:lang="am">ድርገት፡ ሲወርድ፡ ቃጭል፡ መመታቱ፡ ስለ<supplied reason="omitted">፡</supplied> ምን፡ ነው፡ ቢሉ፡ ዮሐንስ፡ ወንጌላዊ፡ ክርስቶስን፡ ሲቀብሩ፡ አይቶ፡ አንዳለቀሰ፡ ነው፡ </explicit>
+                            <textLang mainLang="am"/>       
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i4">
```

Comment:
The cataloguers say that this text is written in another hand. So, this might be technically an additional note. It's difficult to judge without seeing the manuscript. Subsequently, also i5 might be an additional note, because it follows i4, but the cataloguers don't say a word about the scribal hand of i5 or the spatial distribution of the two texts. Sometimes, if the context of the catalogue entry is clear, it is acceptable to move what has been described as a textual unit by the cataloguers to an ‘additional note’, because they had a different perception of what is text and what is an additional note. Here it is not so clear, so I would simply add a note that i4 was written by another hand.